x-pack/test/functional/services/machine_learning/api.ts

```diff
@@ -377,6 +408,36 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       });
     },
 
+    async assertDataFrameAnalyticsJobExist(analyticsId: string) {
```

Comment:
We already have this functionality in this service: `waitForDataFrameAnalyticsJobToExist`, so I would suggest to use that one.,